core/deployment/src/main/java/io/quarkus/deployment/CodeGenContext.java

```diff
@@ -49,6 +52,16 @@ public ApplicationModel applicationModel() {
         return model;
     }
 
+    /**
+     * Current project directory.
+     * The directory would typically be resolved as {@code <project.build.directory>}.
```

Comment:
`project.build.directory` is already there via workDir. What I am asking for is `project.basedir`

Reply:
Ah yes, sorry, I messed up the javadoc.